osbuild/pipeline.py

```diff
@@ -338,7 +335,6 @@ def run(self, store, monitor, libdir, output_directory):
 
             if obj:
                 results = {"success": True}
-                obj.export(output_directory)
```

Comment:
if we initialize `result` with `{"success": True}` in line 326/324, the whole `if` can go and the `else` can become a `if not obj:`, no?